types/order.ts

```diff
@@ -27,4 +33,9 @@ export interface Order {
     shipping_cost_ex_tax: string;
     total_inc_tax: string;
     total_tax: string;
-}
\ No newline at end of file
+}
+
+export interface ShippingOrder {
+    shipping_addresses: BillingAddress[];
```

Comment:
We should define a new type, even if they overlap `ShippingAddress !== BillingAddress`. The types may look and act similar, but they speak different languages.

Reply:
Would it make sense to have a base `Address` type to extend from?

Reply:
Let's keep them separate. It's alright to have some duplication.